lib/datasource/terraform-provider/index.ts

```diff
@@ -60,6 +60,7 @@ async function queryRegistry(
   const currentVersion = dep.releases.find((release) => {
     return res.version === release.version;
   });
+  // istanbul ignore else
```

Comment:
I don't see any `else`

Reply:
yes, but coverage thinks there is one, because we have no test for a falsy if. I don't think adding one makes an improvement here just to cover the non existing else branch